docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
```

Comment:
It's not wizard-like. It's a wizard. Find out its name. It's probably the Create project wizard.

Reply:
No one is around to ask now. I went with *Create project* wizard. The GUI team said they changed it from being a wizard (It is not located in the middle of the screen like the other wizards I'm familiar with) but it has *Next* and *Back* buttons, so who knows.

Reply:
It's a wizard. If that's your question